scripts/core/setup.py

```
@@ -0,0 +1,391 @@
+                self.status["db_pwd_file"] = True
+                break
+                
+        # Check if .env has secrets (basic check)
+        if Path(".env").exists():
+            try:
+                with open(".env", "r") as f:
+                    content = f.read()
+                    if "DJANGO_SECRET_KEY" in content and "DJANGO_SALT" in content:
+                        self.status["secrets_generated"] = True
+            except Exception:
+                pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)